src/app/Components/Pages/contact-us/landing-page-contact-us/landing-page-contact-us.component.css

```diff
@@ -0,0 +1,155 @@
+.container-fluid {
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: url(../../../../../assets/contactus-images/contact-us-landing.png);
+    background-size: cover;
+    background-position: center;
+    overflow: hidden;
+}
+
+.container-fluid h1 {
+    font-weight: 900;
+    font-family: 'Outfit';
+}
+
+.container-fluid h6 {
+    font-family: 'Outfit';
+
+}
+.card{
+    background: none;
+    border: none;
+    width: 490px;
+    height: 120px;
+
+}
+
+.row.mt-4 {
+    display: flex;
+    justify-content: center;
+}
+
+.btn-primary {
+    background-color: var(--secondary-color);
```

Comment:
This primary button have secondary -color why?

Reply:
secondary color is basically a generic color. and it is being used all over the page.